ports/servoshell/desktop/app_state.rs

```diff
@@ -109,6 +109,10 @@ impl RunningAppState {
     pub(crate) fn new_toplevel_webview(self: &Rc<Self>, url: Url) {
         let webview = self.servo().new_webview(url);
         webview.set_delegate(self.clone());
+
+        webview.focus();
+        webview.raise_to_top(true);
+
```

Comment:
do we need to add this to the auxiliary webview code path too?

Reply:
Yep. I've done that now, though this is still a bit broken due to the focus issues we talked about today.